packages/lab/src/stepper-input/StepperInput.tsx

```diff
@@ -51,94 +98,46 @@ export const StepperInput = forwardRef<HTMLDivElement, StepperInputProps>(
       window: targetWindow,
     });
 
-    const adornmentRef = useRef<HTMLDivElement | null>(null);
     const inputRef = useRef<HTMLInputElement | null>(null);
 
     const {
-      decrementButtonDown,
-      getButtonIcon,
       getButtonProps,
       getInputProps,
-      incrementButtonDown,
       isAtMax,
       isAtMin,
-      refreshCurrentValue,
       stepperDirection,
-      valuesHaveDiverged,
     } = useStepperInput(props, inputRef);
 
-    useActivationIndicatorPosition(
-      adornmentRef,
-      valuesHaveDiverged() || showRefreshButton
-    );
-
     const endAdornment: ReactNode = (
-      <div className={withBaseName("adornmentContainer")} ref={adornmentRef}>
+      <>
```

Comment:
the endAdornments seem to be unaccessible via keyboard

Reply:
this aligns with what we have in uitk but may change